rs/artifact_pool/src/ingress_pool.rs

```diff
@@ -349,15 +349,11 @@ impl ValidatedPoolReader<SignedIngress> for IngressPoolImpl {
 
 impl IngressPoolThrottler for IngressPoolImpl {
     fn exceeds_threshold(&self) -> bool {
-        let ingress_count = self.validated.size() + self.unvalidated.size();
-        let ingress_bytes = self.validated.count_bytes() + self.unvalidated.count_bytes();
-
-        if ingress_count >= self.ingress_pool_max_count
-            || ingress_bytes >= self.ingress_pool_max_bytes
-        {
+        if self.validated.peer_counters.exceeds_limit(&self.node_id) {
             self.ingress_messages_throttled.inc();
             return true;
         }
+
         false
     }
 }
```

Comment:
I wonder if the API should not be changed to `exceeds_threshold(&self, node_id: &NodeId)`. With the current usage, this is fine since we only use it in the HTTP endpoint but it is a little surprising that it uses its node id. 

You don't have to change anything just curious what you think.

Reply:
Good point, I will change it in a subsequent PR